crates/extensions/c8y_firmware_manager/src/actor.rs

```diff
@@ -253,13 +253,13 @@ impl FirmwareManagerActor {
                 .is_some()
             {
                 if let Ok(token) = self.message_box.jwt_retriever.await_response(()).await? {
-                    DownloadRequest::new(firmware_url, &cache_file_path)
+                    DownloadRequest::new(firmware_url, cache_file_path.as_std_path())
```

Comment:
```suggestion
                    DownloadRequest::new(firmware_url, cache_file_path.as_ref())
```
Not a request for change but more of an FYI. But, I also know that you prefer explicit conversion functions over the generic ones like `as_ref()` or `into()` 😄